tekton/ci/templates/website-template.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: triggers.tekton.dev/v1alpha1
```

Comment:
This file needs to be added to the `kustomize.yaml` in this folder so that it gets deployed